Header header
string trial_name
string agent
string error


bool impact_flag
bool runComplete_flag

uint8 k_ep
uint8 k_run

uint8 n_rollouts
float32 gamma
float32 h_ceiling

float32[] alpha_mu
float32[] alpha_sigma
float32[] mu
float32[] sigma

float32[] policy

float32 reward
float32 reward_avg

float32[] vel_d
float32[] M_d
bool[] leg_contacts


